### PR TITLE
townhall events

### PR DESCRIPTION
**Pull request by wsbctechnicalblog**

### CODE REVIEW

**Comment by wpschaub** [comment]
@wsbctechnicalblog 
- Can you add the ceremony tag?
- Please also review the townhall working agreement poster and advise of changes.
- Happy to add the poster and download section when you are ready to receive a commit.

**Comment by wsbctechnicalblog** [comment]
@wpschaub I'll open a new, simpler PR (forgot that this one also pushed unnecessary noise with generated HTML files), and I'll make the changes you requested. Thanks.

File: content/townhall.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+Title: Guidelines to start and terminate the Townhall Events
+Date: 2021-11-12
+Category: Posts
+Tags: technical-excellence
```
Comment (wpschaub) [inline content/townhall.md:4]: @wsbctechnicalblog can you add the **ceremony** tag?

File: content/townhall.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+Attendees are responsible for the effective utilization of the time spent in the Townhall, as well as the time spent preparing any material to be presented in the Townhall. Attendees are expected to demonstrate brisk and sustained participation in the Townhall.  
+
+If the Townhall demonstrates effective progress (i.e., identified and prioritized bottlenecks are swiftly being minimized/removed from the system), organizers will be given allowances to demand more time that the staff should dedicate to identifying system-wide bottlenecks and improving the overall system performance. 
+
+## Terminating the Townhall 
+
+Everything good and nice must eventually end. The desired end-state of the system as a whole is a fully flexible, secure, reliable, resilient, and anti-fragile system that perfectly aligns with WorkSafeBC business priorities. Once the system reaches that high level of performance, the Townhall should be terminated. 
+
+There is also a possibility that the Townhall events may not be yielding the hoped-for results. In case of such eventuality, organizers should terminate the Townhall.
```
Comment (wpschaub) [inline content/townhall.md:73]: @wsbctechnicalblog please add overview and townhall pdf download section.

**Review by wpschaub — CHANGES_REQUESTED** [review]

**Comment by wsbctechnicalblog** [comment]
This PR is too messy, replaced it with a streamlined townhall events.

**Pull request closed without merging**